### PR TITLE
Adding react-albus

### PR DESCRIPTION
**Pull request by nathanforce**

### CODE REVIEW

**Comment by nathanforce** [comment]
How do we get @jsdelivrbot to retry?

File: files/react-albus/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "react-albus",
+  "repo": "americanexpress/react-albus",
+  "files": {
+    "basePath": "lib/"
```
Comment (megawac) [inline files/react-albus/update.json]: I don't see the `lib/` folder on the latest tag https://github.com/americanexpress/react-albus/tree/v1.0.1 (perhaps you want to use `npm` as the packageManager?

**Comment by megawac** [comment]
Ah, thats more of just a heads up. It just checks the latest commit once

**Comment by nathanforce** [comment]
@megawac Updated.  Do I still need the files key in update.json when using npm as the packageManager?

**Comment by megawac** [comment]
Yep, cheers

**Pull request merged**